base.py

```diff
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
+    bl_label = "Some of the nodes/elements are missing"
+    
+    hide = bpy.props.BoolProperty()
```

Comment:
Uhm... No actually I like your solution, I just didn't understand it at the beginning. It's okay to let users keep the old objects in place, if they so desire.